src/renderers/dom/shared/ReactDOMInstrumentation.js

```diff
@@ -0,0 +1,16 @@
+/**
+ * Copyright 2013-2015, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule ReactDOMInstrumentation
+ */
+
+'use strict';
+
+var ReactDOMDebugTool = require('ReactDOMDebugTool');
+
+module.exports = {debugTool: ReactDOMDebugTool};
```

Comment:
What is the difference between “debug tool” and a “dev tool”? I got tripped by this a couple of times because I write a `Devtool` but I’m then adding it to a `DebugTool` which I refer to as `ReactDOMInstrumentation.debugTool`. Why is there more than one layer of indirection?